src/Renderer.ts

```diff
@@ -504,13 +504,11 @@ export default class Renderer {
     sprite: Sprite | Stage,
     options: RenderSpriteOptions
   ): void {
-    const spriteScale = "size" in sprite ? sprite.size / 100 : 1;
-
     this._renderSkin(
       this._getSkin(sprite.costume),
       options.drawMode,
       this._getDrawable(sprite).getMatrix(),
-      spriteScale,
+      this._getDrawable(sprite).getSpriteScale(),
```

Comment:
Maybe move `this._getDrawable(sprite)` into a variable now that we're calling it twice? More for code style than any micro-optimization concerns.